data/labels.json

```diff
@@ -4,15 +4,15 @@
     "title": "Indy",
     "color": "orange",
     "items_ids": [
-      141
+
     ]
   },
   {
     "id": 417,
     "title": "Romance",
     "color": "blue",
     "items_ids": [
-      817
+
```

Comment:
Please make sure this information is in this file, and stays there when the program loads and closes